python/modules/IcePy/ValueFactoryManager.cpp

```diff
@@ -110,7 +98,16 @@ IcePy::ValueFactoryManager::add(PyObject* valueFactory, string_view id)
 {
     try
     {
-        add(make_shared<FactoryWrapper>(valueFactory), id);
+        CustomValueFactoryPtr f = make_shared<CustomValueFactory>(valueFactory);
```

Comment:
Do we need to create the CustomValueFactory here, or can we do it when we insert?

Reply:
Switched to try_emplace, which is even more correct.